### PR TITLE
Add telemetry

### PR DESCRIPTION
**Pull request by ChemiKyle**
Addresses #60 without a solution in place for running in docker with an additional `.env` file.

### CODE REVIEW

**Comment by ChemiKyle** [comment]
Oddly, `docker run -ti --rm --env-file=.env --env-file=add.env tt_engine bash` then running `env` shows the overloaded `EMAIL_TO` properly in place.

**Comment by ChemiKyle** [comment]
... I forgot you have to `build` again and had a file named `test.R` in my old image.

File: examples/tt_engine_prod.cron [hunk 1]
```diff
@@ -9,3 +9,6 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 
 # Send cleared for campus flags generally every 4 hours
 50 0,4,8,12,16,20 * * * root /usr/bin/docker run -v /ctsit/tt_engine/oneuftesting_ctsit.key:/root/.ssh/id_rsa --rm --env-file /ctsit/tt_engine/prod.env tt_engine Rscript send_cleared_status_to_person_hub.R
+
+# Load laboratory results into REDCap at 8:15 AM and 12:15 PM, overloading env-file with extra recipients of an email digest
+15 8,12 * * * root /usr/bin/docker run -v /ctsit/tt_engine/oneuftesting_ctsit.key:/root/.ssh/id_rsa  --rm --env-file /ctsit/tt_engine/prod.env --env-file /ctsit/tt_engine/to_line_for_write_lab_results_to_results_project.env tt_engine Rscript write_lab_results_to_results_project.R
```
Comment (ChemiKyle) [inline examples/tt_engine_prod.cron:14]: >This job should run at 0800 and 1215.

Maybe my cron-fu is low level but I can't think of a way to do this in a oneliner so this is set to run at 8:15 and 12:15.

Comment (pbchase) [reply examples/tt_engine_prod.cron:14]: looks good to me

**Comment by pbchase** [comment]
I like it. I had to revise some comment text, but I think this will allow us to automate.

**Pull request merged**